### DISCUSSION TITLE
📸 Showcase your unique GitHub Profile

### DISCUSSION THREAD

**Discussion by x0rzavi** [x0rzavi/github-readme-terminal#2] [Show and tell]
Please post a 🔗 _**link to your GitHub profile**_ in addition to a 🖼️ _**preview image/video**_. If you are comfortable sharing the corresponding file the image/video has been generated from, please share that too ☺️. Happy Hacking ✨

**Comment by lewisdoesstuff** [comment]
Thanks a bunch for this great little library!  
Here's my first attempt, taking _some_ (a lot) of inspiration from yours :)

https://github.com/lewisdoesstuff

[Source](https://github.com/lewisdoesstuff/lewisdoesstuff)

**Comment by kaguya3311** [comment]
This library is awesome, i took a bit from lewis's thing, but put my own spin on it, openrc edition

https://github.com/reggietorres

**Comment by Bryan-Campuzano** [comment]
tnks a lot for the library, before some headaches i got this:

https://github.com/Bryan-Campuzano

tnk u dude.

**Comment by grosheth** [comment]
Thanks for the library, it's really cool!

I tried to make a [bottom](https://github.com/ClementTsang/bottom) replica!

https://github.com/grosheth

![output](https://github.com/user-attachments/assets/64b93a20-e20c-42fd-9a39-9567cb7ee830)

**Comment by Ethorbit** [comment]
Really fun to play with!

https://github.com/Ethorbit
I want to add more to it, but this is what I have so far

**Comment by lynkos** [comment]
Super cool project!

https://github.com/lynkos

![Generated stats gif](https://raw.githubusercontent.com/lynkos/lynkos/refs/heads/main/output.gif)

[Source](https://github.com/lynkos/lynkos/blob/main/main.py)
